File: angular-eventsAndTickets/src/app/DTOs/eventDTO.ts
import { ListKeyManager } from '@angular/cdk/a11y';
import { eventTypeDTO } from './eventTypeDTO';

export interface eventDTO {
  Id: number;
  Name: string;
  TicketPrice: number;
  Description: string;
  City: string;
  Date: Date;
  Poster: string;
}

File: EventsAndTicketsAPI/EventsAndTicketsAPI/DTOs/EventDTO.cs
using EventsAndTicketsAPI.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EventsAndTicketsAPI.DTOs
{
    public class EventDTO
    {
        public int Id { get; set; }
        public int TypeId { get; set; }
        public string Name { get; set; }
        public int TicketPrice { get; set; }
        public string Description { get; set; }
        public string City { get; set; }
        public DateTime Date { get; set; }
        public string Poster { get; set; }
        public List<EventAndEventType> EventAndEventTypes { get; set; }
        public List<EventTypeDTO> EventTypeDto { get; set; }
    }
}
